src/main/kotlin/day15/comment/repository/CommentRepositoryImpl.kt

```
@@ -0,0 +1,18 @@
+import org.hyunjung.day15.comment.datasource.CommentDataSource
+import org.hyunjung.day15.comment.model.Comment
+
+class CommentRepositoryImpl(
+    private val remoteDataSource: CommentDataSource,
+    private val mockDataSource: CommentDataSource? = null
+) : CommentRepository {
+
+    override suspend fun getComments(postId: Int): List<Comment> {
+        return try {
+            remoteDataSource.getComments(postId)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)